app/src/main/java/nextstep/signup/MainActivity.kt

```diff
@@ -29,36 +50,128 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    //Greeting("Android")
+                    SignUpScreen()
                 }
             }
         }
     }
 }
 
 @Composable
-fun HelloView(
-    name: String,
-    modifier: Modifier = Modifier,
+fun SignUpScreen() {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = Color.White)
+            .padding(horizontal = 32.dp),
+    ) {
+        val (userName, setUserName) = remember { mutableStateOf("") }
+        val (email, setEmail) = remember { mutableStateOf("") }
+        val (password, setPassword) = remember { mutableStateOf("") }
+        val (passwordConfirm, setPasswordConfirm) = remember { mutableStateOf("") }
+
+        Spacer(modifier = Modifier.height(60.dp))
+        Text(
+            text = stringResource(R.string.signup_header_title),
+            color = Color.Black,
+            fontSize = 26.sp,
+            fontFamily = RobotoBold,
+        )
+        Spacer(modifier = Modifier.height(42.dp))
+        SignUpTextField(
+            label = stringResource(R.string.signup_user_name),
+            onTextChanged = { setUserName(it) },
+            text = userName,
```

Comment:
여기는 알고 계시겠지만, 람다의 도움을 받지 않고 바로 세팅할 수도 있답니다!
```suggestion
            onTextChanged = setUserName,
            text = userName,
```